appengine/standard/ndb/schema_update/update_schema.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import updated_picture_model
+#import models
+from google.appengine.ext import deferred
+from google.appengine.ext import ndb
+
+BATCH_SIZE = 100  # ideal batch size may vary based on entity size.
+
+class Picture(ndb.Model):
+    author = ndb.StringProperty()
+    name = ndb.StringProperty(default='')  # Unique name.
+    #num_votes = ndb.IntegerProperty(default=0)
+    #avg_rating = ndb.FloatProperty(default=0)
+
+def get_current_entities():
+    current_entities = list(Picture.query().fetch())
+    return current_entities
+
+def add_entity(author_value, name_value):
+    new_pic = Picture(author=author_value, name=name_value)
+    new_pic.put()
+        
+
+def UpdateSchema(cursor=None, num_updated=0):
+    Picture = updated_picture_model.Picture()
+    query = Picture.query()
+    if cursor:
```

Comment:
You can just consolidate this as

```
data, cursor, more = query.fetch_page(BATCH_SIZE, start_cursor=cursor)
```

if cursor is None, it's the same as it not being specified.